pkgs/tools/security/gopass/git-credential.nix

```diff
@@ -21,7 +23,14 @@ buildGoModule (finalAttrs: {
 
   subPackages = [ "." ];
 
-  nativeBuildInputs = [ makeWrapper ];
+  nativeBuildInputs = [
+    makeWrapper
+    writableTmpDirAsHomeHook
+  ];
+
+  buildInputs = [
+    gopass
+  ];
```

Comment:
What is this for?

Reply:
IIUC, `gopass` is a runtime dep of the wrapped executable, so putting it in `buildInputs` ensures the reference is captured in the output closure.

Reply:
> IIUC, `gopass` is a runtime dep of the wrapped executable, so putting it in `buildInputs` ensures the reference is captured in the output closure.

Here OTH it is a bit different. Technically you should know there's no difference with `gopass` in `buildInputs` or not, and it doesn't make a difference in terms of output/ build closure size.

Reply:
I’ve just understood what “Automatic Runtime Dependencies” are as described in Nix Pill 9. Thanks a lot for your explanation!